src/roundedEquals.js

```diff
@@ -4,11 +4,10 @@ define(function() {
    *
    * @access private
    * @function roundedEquals
-   * @param {number} a
-   * @param {number} b
-   * @returns {boolean}
+   * @param {number} a - first integer
+   * @param {number} b - second integer
+   * @returns {boolean} true if the first integer is within 1 of the second
```

Comment:
wording is ambiguous in the falsey case

Reply:
not sure how to word it better than this, maybe you can add your two cents